recipes-demos/qmltermwidget/qmltermwidget.bb

```diff
@@ -0,0 +1,21 @@
+# Recipe taken from https://github.com/webOS-ports/meta-webos-ports
+# Commit: 528204c26b1550f54b7b93cbc9c32352ab5b0839
+SUMMARY = "A terminal emulator QML widget, based on LXQt's QTermWidget"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=4641e94ec96f98fabc56ff9cc48be14b"
+
+PV = "0.14.1+git${SRCPV}"
```

Comment:
SRCPV is dropped and no longer valid in scarthgap.

Ref,

- https://git.openembedded.org/openembedded-core/commit/?id=a8e7b0f932b9ea69b3a218fca18041676c65aba0
- https://git.ti.com/cgit/arago-project/meta-ti/commit/?h=scarthgap&id=d7624a767e20c776eb937c2942127299b4dc0093